beacon.yaml

```
@@ -612,10 +615,10 @@ components:
             URL to an external system, such as a secured beacon or a system
             providing more information about a given allele (RFC 3986 format).
         info:
-          description: 'Additional structured metadata, key-value pairs.'
-          type: array
-          items:
-            $ref: '#/components/schemas/KeyValuePair'
+          description: 'Additional metadata about the response.'
+          type: object
+          example:
+            additionalVariants: true
```

As this field is totally free, examples should be generic (this one is confusing to me):
```suggestion
            additionalInfoKey: additionalInfoValue
```